test/CMakeLists.txt

```diff
@@ -18,19 +18,19 @@ add_library(pcl_gtest STATIC ${GTEST_SRC_DIR}/src/gtest-all.cc)
 
 enable_testing()
 
-set(PCL_CTEST_COMMAND "${CMAKE_CTEST_COMMAND}")
+set(PCL_CTEST_ARGUMENTS)
```

Comment:
I think this is not required actually. If the variable does not exists I think it will only be empty.